### DISCUSSION TITLE
Firewall issues installing .net interactive

### DISCUSSION THREAD

**Discussion by juanbstevens** [dotnet/interactive#2787] [General]
Is there any way to download all dependencies and install manually? I'm running into issues trying to install through my federal agency's firewall. I download the extension and able to install but run into issues when the extension tries to install the .net interactive version 1.0.415102. It just sits there and says installing.

**Comment by jonsequitur** [comment]
This is a known issue: #2601. A possible workaround just occurred to me, which I'll post there.

**Comment by juanbstevens** [comment]
Ok thank you. I just read through that thread. Yeah, this really sucks for all of us federal employees who want to use the tools.

________________________________
